app/assets/javascripts/discourse/app/components/topic-list.js

```diff
@@ -190,4 +190,24 @@ export default Component.extend(LoadMore, {
       this.rerender();
     });
   },
+
+  keyDown(e) {
+    if (e.keyCode === 13 || e.keyCode === 32) {
```

Comment:
slightly outside the scope of this PR, but I think we should stop using keyCode and use code. So not sure we should keep introducing this kind of code.

See:
- https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
- https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code